src/python/gudhi/cubical_complex.pyx

```diff
@@ -145,6 +146,33 @@ cdef class CubicalComplex:
             persistence_result = self.pcohptr.get_persistence(homology_coeff_field, min_persistence)
         return persistence_result
 
+    def cofaces_of_persistence_pairs(self):
```

Comment:
I am wondering if having no way to know the dimension of the feature is going to be a problem.

Reply:
As you noted below, I actually include the dimension, I just forgot to update the doc ;-)